client/signup/steps/theme-selection/index.jsx

```diff
@@ -22,14 +26,20 @@ module.exports = React.createClass( {
 		signupDependencies: React.PropTypes.object.isRequired,
 	},
 
+	getInitialState() {
+		return {
+			previewTheme: null,
+		};
+	},
+
 	getDefaultProps() {
 		return {
 			useHeadstart: true,
 		};
 	},
 
-	handleScreenshotClick( theme ) {
-		var themeSlug = theme.id;
+	pickTheme( theme ) {
+		let themeSlug = theme.id;
```

Comment:
It looks like this variable doesn't get reassigned, so it could probably be `const`.